qt/HelloQt.cpp

```diff
@@ -0,0 +1,45 @@
+#include "HelloQt.hpp"
+#include <QHBoxLayout>
+#include <QVBoxLayout>
+#include <QPushButton>
+#include <QLabel>
+#include <QDebug>
+
+HelloQt::HelloQt( QWidget* parent )
+	:QWidget( parent )
+{
```

Comment:
The style commonly used in Qt is to have tight brackets, e.g. (QWidget* parent) instead of ( QWidget* parent )
They also tend to put the asterisk on the value side in Qt examples, but I agree with you here it should be on the type.